src/System.Private.CoreLib/shared/System/Runtime/CompilerServices/DateTimeConstantAttribute.cs

```diff
@@ -14,6 +15,6 @@ public DateTimeConstantAttribute(long ticks)
             _date = new DateTime(ticks);
         }
 
-        public override object Value => _date;
+        public override object? Value => _date; // TODO-NULLABLE: https://github.com/dotnet/roslyn/issues/23268
```

Comment:
We're already supressing the warning you're referring to in the issue, why is this needed?

Reply:
Because without this we get a warning:
```
shared\System\Runtime\CompilerServices\DateTimeConstantAttribute.cs(18,32): error CS8608: Nullability of reference types in type doesn't match overridden member. [d:\repos\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
```
As @cston commented at https://github.com/dotnet/corert/pull/7276#issuecomment-481717253, "`#nullable enable` will re-enable any nullable warnings that were disabled in the project file".

Reply:
😢 ok makes sense.